source/Nevermore/Advanced/SelectBuilderBase.cs

```diff
@@ -0,0 +1,192 @@
+﻿using System;
```

Comment:
These Select Builders have been given room to breath in their own files, instead of all being squished into the single file